### PR TITLE
kvrpc: add cmd BatchRollback, update CommitResponse

### PR DESCRIPTION
**Pull request by disksing**
@morefreeze @ngaut

### CODE REVIEW

File: proto/kvrpcpb.proto [hunk 1]
```diff
@@ -16,6 +16,7 @@ enum MessageType {
     CmdRollbackThenGet  = 6;
     CmdCommitThenGet    = 7;
     CmdBatchGet         = 8;
+    CmdRollback         = 9;
```
Comment (morefreeze) [inline proto/kvrpcpb.proto]: How about `CmdBatchRollback` ?

**Comment by disksing** [comment]
@morefreeze Updated.

**Comment by morefreeze** [comment]
LGTM

**Comment by ngaut** [comment]
LGTM

**Pull request merged**